src/KurrentDB.Core/Services/SubscriptionsService.cs

```diff
@@ -307,6 +309,30 @@ private static Message CloneReadRequestWithNoPollFlag(Message originalRequest) {
 		ReissueReadsFor(message.Event.EventStreamId, message.CommitPosition, message.Event.EventNumber);
 	}
 
+	async ValueTask IAsyncHandle<StorageMessage.SecondaryIndexRecordCommitted>.HandleAsync(StorageMessage.SecondaryIndexRecordCommitted message, CancellationToken token) {
+		_lastSeenSecondaryIndexLogPosition = message.LogPosition;
+
+		var streamCategory = SystemStreams.GetStreamCategory(message.Event.EventStreamId ?? message.Event.EventStreamId);
```

Comment:
```suggestion
		var streamCategory = SystemStreams.GetStreamCategory(message.Event.EventStreamId ?? message.Event.EventStreamId);
```

`message.Event.EventStreamId` is specified twice